src/SprykerEco/Yves/Payone/Controller/IndexController.php

```diff
@@ -240,53 +219,6 @@ public function cancelRedirectAction(Request $request)
         return $this->redirectResponseInternal(static::ROUTE_CHECKOUT_PAYMENT);
     }
 
-    /**
-     * @param \Symfony\Component\HttpFoundation\Request $request
-     *
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    public function glueCancelRedirectAction(Request $request): Response
-    {
-        $orderReference = $request->query->get('orderReference');
-        $urlHmac = $request->query->get('sig');
-
-        if ($orderReference) {
```

Comment:
I'm not 100% sure right now if that's sufficient. Maybe it's better to test for a specific value instead of just the truthiness of the variable. What happens if the value doesn't exist in the query? Either you should provide a default value or check if the value exists with $request->query->has().

Reply:
I am removing the lines as we no longer need them